doc/scaling_spire.md

```diff
@@ -1,43 +1,92 @@
 # Scalability
 
-A SPIRE deployment is composed of a number one or more SPIRE Servers that share a replicated datastore, and one or more SPIRE Agents. With only one SPIRE Server per SPIRE deployment, the memory and CPU usage tends to grow proportionally with the number of workloads in the deployment. To support very large numbers of workloads and agents within a given trust domain (tens of thousands or hundreds of thousands of nodes), the number of SPIRE Servers can be scaled horizontally. With multiple servers, the load is distributed. In addition to capacity, multiple SPIRE Servers can be configured for high availability.
+A SPIRE deployment has the capacity to be changed in size or scale to accommodate a growing amount of workloads. A SPIRE deployment is composed of a number of one or more SPIRE Servers that share a replicated datastore, or conversely, a set of SPIRE servers in the same trust domain, and at least one SPIRE Agent, but typically more than one.
 
-It is important to note when scaling the number servers horizontally, be it for high availability or load distribution purposes, that in any configuration of two or more SPIRE Servers, all servers share the same datastore. Each server maintains its own CA however, which may be self-signed or an intermediate off of a shared root authority (i.e. when configured with an UpstreamAuthority).
+Deployments range in size. A single SPIRE Server may accommodate a number of Agents and Workload Registration entries. A scale sizing consideration is that with one SPIRE Server per SPIRE deployment, the memory and CPU consumption tends to grow proportionally to the number of Workloads in the deployment. A single SPIRE Server also represents a single point of failure.
```

Comment:
"A scale sizing consideration is that with one SPIRE Server per SPIRE deployment, the memory and CPU consumption tends to grow proportionally to the number of Workloads in the deployment."

Is this not also true when more than instance of SPIRE Server is deployed?

Reply:
Worth calling out _why_ this consumption grows (I presume it's because of the increased number of singing operations) proportional to the number of workloads. Are there other variables likely to affect this too? (eg. the TTL of the SVIDs?)